### ISSUE TITLE
[FR] Clock

### ISSUE THREAD

**Issue by jlucasgithub** [dwainscheeren/dwains-lovelace-dashboard#76]
Hi

Small feature request. Is it possible to have big clock if weather is not configured?

Thanks

**Comment by Tosko4** [comment]
It is possible, but we’ll have to wait for @dwainscheeren to acknowledge these FR’s and assign a future release tag to them

**Comment by jlucasgithub** [comment]
Hi @dwainscheeren 
I am just wondering what do you think about this? Is it going to happen in next release maybe 1.5?

**Comment by JurajNyiri** [comment]
+1 on Clock. I can code this in if needed and open pull request. Would this be preferred?

**Comment by dwainscheeren** [comment]
Clock is banned

**Comment by JurajNyiri** [comment]
Ok I will then fork the repo and keep the changes in my fork only. Thank you!

**Comment by dwainscheeren** [comment]
If you want you can post a link to the fork here with a screenshot, that way I can look how you (and others) got it in mind. I'm working to put an clock in 3.0, but not anymore in 2.*.

**Comment by JurajNyiri** [comment]
![Screenshot_2021-12-28-18-39-06-210](https://user-images.githubusercontent.com/5785999/147592622-066742c6-357f-448e-9d4c-7648ac41ad98.jpeg)

[In this commit](https://github.com/JurajNyiri/dwains-lovelace-dashboard/commit/1659c1481aa80e53269bc424079343783c68c529) is the implementation. Linter did modify a couple more lines, but the relevant change is [here](https://github.com/JurajNyiri/dwains-lovelace-dashboard/commit/1659c1481aa80e53269bc424079343783c68c529#diff-2084b79c89ec813f78794c8136ec0bea78c583271fdac2839345c7847a073353R163-R177). It basically just replaces "Good evening" text. It could also be done cleaner (time passed from parent) but since its just my fork I did not bother.

I also changed a couple more things in another commit about the cards and design as I wanted a bit different look - removed big weather card (used small instead on all screens), removed big alarm button (used small instead on all screens), removed House Information button (I saw no use for me) and made it all centered. Finally I played a bit with margins and paddings on the header. This is out of scope of this FR, just wanted to clarify for anyone looking at screenshot and wondering.
